backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/database/entity/VersionEntity.kt

```diff
@@ -0,0 +1,20 @@
+package com.tngtech.apicenter.backend.connector.database.entity
+
+import org.hibernate.search.annotations.ContainedIn
+import org.hibernate.search.annotations.Field
+import org.springframework.stereotype.Indexed
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.ManyToOne
+
+@Entity
+@Indexed
+data class VersionEntity(
+    @Id @GeneratedValue val id: UUID,
```

Comment:
Why does it need a UUID? It should be uniquely identifiable by spec and version.

Reply:
Because we need to address Versions directly from the frontend and I found unique IDs to come in handy here.

Reply:
Hm. I'm not sure if this isn't more confusing, since we now have two things that are addressable by UUIDs. Maybe a simple concatenation of Spec-UUID + Version would work?

Reply:
How about a query like `GET /specifications/{specificationId}?version={version}` 
or `GET /specifications/{specificationId}/{version}`
?